election-ui-kotlin/settings.gradle

```diff
@@ -5,6 +5,10 @@ pluginManagement {
         mavenCentral()
     }
 }
+
+plugins {
+    id 'org.gradle.toolchains.foojay-resolver-convention' version '0.8.0'
+}
```

Comment:
I don't think this is needed since we're not using `jvmToolchain` anywhere.

Reply:
yep, good catch, dropped by an errant wizard